File: week1-python/Getting_started_with_images/Color_Images.py
# Import libraries
import cv2
import numpy as np
from dataPath import DATA_PATH

# Path of the image to be loaded
# Here we are supplying a relative path
imagePath = DATA_PATH + "/images/musk.jpg"

# Read the image
img = cv2.imread(imagePath)
print("image Dimension ={}".format(img.shape))

File: week1-cpp/Getting_started_with_images/Color_Images.cpp
#include <iostream>
#include <opencv2/opencv.hpp>
#include <opencv2/core.hpp>
#include <opencv2/highgui.hpp>
#include "dataPath.hpp"

using namespace std;
using namespace cv;

int main(void)
{
	string imagePath = DATA_PATH + "/images/musk.jpg";
	// Read the image
	Mat img = imread(imagePath);

	cout << "image size = " << img.size();
	return 0;
}	
